README.md

```diff
@@ -2,16 +2,17 @@
 
 # Ibex RISC-V Core
 
-Ibex is a small and efficient, 32-bit, in-order RISC-V core with a 2-stage pipeline that implements
-the RV32IMC instruction set architecture.
-
-<p align="center"><img src="doc/03_reference/images/blockdiagram.svg" width="650"></p>
```

Comment:
I would not remove the block diagram because it's the only figure we have in the readme and I would argue that in the meantime it has gained quite some recognition value. It may not accurately represent the configuration with the writeback stage/BT ALU. But if we wanted to accurately represent this configuration, how would this new block diagram look like? On register in the line between LSU and RF?

Reply:
If we want to be really accurate we can also put a caption and state that this figure shows the "small" configuration (according to the verification status still the most important one).